dev-manuals/docker-compose/docker-compose-unity.md

```diff
@@ -0,0 +1,18 @@
+# Use Docker Compose files with unity
+
+The usage of [Docker Compose files](docker-compose.md) with unity projects differs from our other projects, cause we have no ability to build the project in a docker container. Cause of that ther exists no `docker-compose.yaml`.
```

Comment:
```suggestion
The usage of [Docker Compose files](docker-compose.md) with unity projects differs from our other projects, because we have no ability to build the project in a docker container. Because of that there exists no `docker-compose.yaml`.
```